pkg/agent/manager/cache/cache_test.go

```diff
@@ -436,20 +437,24 @@ func TestCheckSVIDCallback(t *testing.T) {
 	// no calls because there are no registration entries
 	cache.Update(&Update{
 		Bundles: makeBundles(bundleV2),
-	}, func(entry *common.RegistrationEntry, svid *X509SVID) {
+	}, func(existingEntry, newEntry *common.RegistrationEntry, svid *X509SVID) {
 		assert.Fail(t, "should not be called if there are no registration entries")
 	})
 
-	foo := makeRegistrationEntry("FOO")
+	foo := makeRegistrationEntry("FOO", 60)
```

Comment:
nit: it seems preferable to create another method (i.e. `makeRegistrationEntryWithTTL`) or something that allowed you to use a different TTL at this one spot than changing the existing function and passing the default `ttl` everywhere else...